Sources/SwiftFormat/Utilities/URL+isRoot.swift

```diff
@@ -12,21 +12,43 @@
 
 import Foundation
 
+#if os(Windows)
+import WinSDK
+#endif
+
 extension URL {
+  /// Returns a `Bool` to indicate if the given `URL` leads to the root of a filesystem.
+  /// A non-filesystem type `URL` will always return false.
   @_spi(Testing) public var isRoot: Bool {
+    guard isFileURL else { return false }
+
+    #if os(macOS)
+    return self.path == NSOpenStepRootDirectory()
+    #endif
+
+    #if compiler(>=6.1)
     #if os(Windows)
-    // FIXME: We should call into Windows' native check to check if this path is a root once https://github.com/swiftlang/swift-foundation/issues/976 is fixed.
+    return self.path.withCString(encodedAs: UTF16.self, PathCchIsRoot)
+    #elseif os(Linux)
```

Comment:
This should be `#else`. Otherwise this function doesn’t have a return value eg. on platforms that are neither macOS, Linux, nor Windows, like Android.

Reply:
I had that thought and will fix it.